yarn-project/noir-protocol-circuits/src/crates/types/src/abis/private_circuit_public_inputs.nr

```diff
@@ -108,6 +69,9 @@ impl Serialize<PRIVATE_CIRCUIT_PUBLIC_INPUTS_LENGTH> for PrivateCircuitPublicInp
         for i in 0..MAX_READ_REQUESTS_PER_CALL{
             fields.extend_from_array(self.read_requests[i].serialize());
         }
+        for i in 0..MAX_NULLIFIER_KEY_VALIDATION_REQUESTS_PER_CALL{
```

Comment:
Do you know how this missing did not cause a failure? Was it only used for the hash?